code/modules/mob/living/carbon/human/species_types/lizardpeople.dm

```diff
@@ -153,14 +153,26 @@
 /datum/species/lizard/create_pref_temperature_perks()
```

Comment:
Why did you put this in `create_pref_temperature_perks`?

Reply:
because I'm a FOOL and can't read